conviva/src/main/java/com/bitmovin/analytics/conviva/ConvivaAnalyticsIntegration.java

```diff
@@ -681,14 +569,8 @@ public void setSeekEnd() {
         @Override
         public void onEvent(PlayerEvent.AdStarted adStartedEvent) {
             Log.d(TAG, "[Player Event] AdStarted");
-            Client.AdPosition adPosition = AdEventUtil.parseAdPosition(adStartedEvent, bitmovinPlayer.getDuration());
```

Comment:
Is this Ad metadata like Ad position/duration etc. needed to passed into ConvivaAdAnalytics?

Reply:
This is not needed at present.

Reply:
ok. thanks.